app/assets/javascripts/admin.js

```diff
@@ -24,7 +24,8 @@ var assignmentsReady = function(){
 
 }
 
-$(document).ready(assignmentsReady)
+$(document).ready(assignmentsReady);
+$(document).on('page:load', assignmentsReady);
```

Comment:
@okeefm sorry I missed your comment. It is impossible for those two events to both fire on the same page. This is confirmed in the comments on the top answer on the Stack Overflow post that is linked in the PR description